modules/drivers/robosense/BUILD

```diff
@@ -0,0 +1,62 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "librobosense_driver_component.so",
+    linkopts = ["-shared"],
```

Comment:
Please change Line 8 to `linkshared = True,` . Otherwise, compilation will fail with the following message

```
(05:11:32) ERROR: /apollo/modules/drivers/robosense/BUILD:6:10: Linking of rule '//modules/drivers/robosense:librobosense_driver_component.so' failed (Exit 1)
/usr/lib/gcc/x86_64-linux-gnu/7/../../../x86_64-linux-gnu/Scrt1.o: In function `_start':
(.text+0x20): undefined reference to `main'
collect2: error: ld returned 1 exit status
```